### ISSUE TITLE
Map features 

### ISSUE THREAD

**Issue by lordyavin** [theCrag/mobile#3]
[ ] find me button
[ ] display crags
[ ] discover nearby crags

**Comment by nicHoch** [comment]
yes there are many many nice feature to come. But most likely not in version 1.0.

**Comment by lordyavin** [comment]
Are you sure? If such features are missing I won't release it as a the first major version. But that's up to you. Think over it. Start with 0.1.0 ;-)

**Comment by nicHoch** [comment]
yes you are right 0.1 is the new 1.0.
What we target first is not a major release.
